pkgs/development/python-modules/yaspin/default.nix

```diff
@@ -30,14 +31,16 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [
+    pytest-xdist
     pytestCheckHook
   ];
 
-  postPatch = ''
-    # https://github.com/pavdmyt/yaspin/pull/212
-    substituteInPlace pyproject.toml \
-      --replace 'termcolor-whl = "1.1.2"' 'termcolor = "*"'
-  '';
+  disabledTestPaths = [
+    # AssertionError: assert False
+    #  +  where False = <built-in method startswith of str object at 0x7ffff3f0b330>('\r\x1b')
+    #  +    where <built-in method startswith of str object at 0x7ffff3f0b330> = '\r/ '.startswith
```

Comment:
let's open an issue upstream and link to it in the comment here

Reply:
No need to do that, I figured it out on my own. We are using the dumb TERM in the builder and termcolor deactivates colored output with that. \x1b is \033 which is the standard unix color escape code used in colors.

Reply:
ohhh ok nice find -- let's leave a comment to that effect then

Reply:
Already added that. Otherwise we will forget pretty fast why we set that env. 😂